### PR TITLE
Fix capitol image

### PR DESCRIPTION
**Pull request by homosapien-lcy**
After the capitol.jpg moved, chaco/examples/user_guide/plot_types/create_plot_snapshots is broken, use importlib.resources to find the image for loading to fix this

### CODE REVIEW

**Comment by homosapien-lcy** [comment]
closes #879

**Review by corranwebster — CHANGES_REQUESTED** [review]
This breaks on 3.7 and 3.8.

See https://github.com/enthought/pyface/blob/0fb8373b4dbb5e104d3120e88590dbf964865245/pyface/tests/test_pil_image.py#L14-L20 for an example of how to handle this sort of use of importlib across many Python versions

File: examples/user_guide/plot_types/create_plot_snapshots.py [hunk 1]
```diff
@@ -434,9 +434,11 @@ def get_image_plot():
 
 
 def get_image_from_file():
-    import os.path
+    from importlib import resources
 
-    filename = os.path.join("..", "..", "demo", "basic", "capitol.jpg")
+    filename = resources.files(
```
[the remark this answers is not in the document]
Comment (homosapien-lcy) [reply examples/user_guide/plot_types/create_plot_snapshots.py]: Thanks for the tip, just add both importlibs

File: examples/user_guide/plot_types/create_plot_snapshots.py [hunk 2]
```diff
@@ -434,9 +434,18 @@ def get_image_plot():
...
+    # importlib_resources package.
+    try:
+        from importlib.resources import files
+    except ImportError:
+        from importlib_resources import files
+
+    filename = files(
+        'chaco.examples.demo.basic'
+    ).joinpath('capitol.jpg')
     image_source = ImageData.fromfile(filename)
```
Comment (corranwebster) [inline examples/user_guide/plot_types/create_plot_snapshots.py:449]: Typically you want to have this in a with statement something like:
```
resource = files(...)
with as_file(resource) as filename:
    ...
```

**Review by corranwebster — CHANGES_REQUESTED** [review]
Can you please add an appropriate dependency to `importlib_resources` for Python < 3.9 here: https://github.com/enthought/chaco/blob/main/chaco/__init__.py#L19

Also a usage issue for `importlib.resources.files`.

**Comment by homosapien-lcy** [comment]
> Can you please add an appropriate dependency to `importlib_resources` for Python < 3.9 here: https://github.com/enthought/chaco/blob/main/chaco/__init__.py#L19
> 
> Also a usage issue for `importlib.resources.files`.

Got it, just add a "check python3.9 then install importlib_resources" there.
Just want to clarify, what do you mean by usage issue for `importlib.resources.files`?

**Review by corranwebster — APPROVED** [review]

**Comment by corranwebster** [comment]
> Just want to clarify, what do you mean by usage issue for `importlib.resources.files`?

Meaning it should be used in a `with` statement, which you have now done.

This should be good to merge.

**Comment by homosapien-lcy** [comment]
> > Just want to clarify, what do you mean by usage issue for `importlib.resources.files`?
> 
> Meaning it should be used in a `with` statement, which you have now done.
> 
> This should be good to merge.

Thanks Corran! The merge is still block by the ci test, the good old RuntimeError Numpy... Is there anyway we can get around it?

```
RuntimeError: NumPy was built with baseline optimizations: 
(SSE SSE2 SSE3 SSSE3 SSE41 POPCNT SSE42 AVX F16C AVX2) but your machine doesn't support:
(AVX2).
```

**Pull request merged**